azure-pipelines/build/linux/du/templates/doclient-lite-native-steps.yml

```diff
@@ -82,16 +82,3 @@ steps:
     ArtifactName: 'deliveryoptimization-agent'
     publishLocation: 'Container'
 
-# TODO(jimson) Azure artifacts are no longer free to publish to, this task will fail as a result, bug to resolve issue here:
```

Comment:
Im removing these stubs, they're in the commit history if we need to bring them back. Also since you turned on subscription billing for our ADO project, we can theoretically use this feature now.

Nick/Oscar's team own testing going forward, so if they want to use ADO package feeds to retrieve artifacts, we can bring this back.